app/component-library/components-temp/KeyValueRow/KeyValueLabel/KeyValueLabel.tsx

```diff
@@ -54,7 +54,7 @@ const KeyValueRowLabel = ({ label, tooltip }: KeyValueRowLabelProps) => {
         <ButtonIcon
           size={tooltip.size ?? TooltipSizes.Md}
           iconColor={IconColor.Muted}
```

Comment:
```suggestion
          iconColor={IconColor.Alternative}
```
This makes the colors more consistent on the page. Muted is a little hard to perceive, and it fails accessibility contrasts.